scripts/verify.sh

```diff
@@ -14,20 +14,14 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-set -e
```

Comment:
Any reason for removing this?

Reply:
Set e makes it return when first fails but there is only 1 test from bash perspective, right?